src/display/text_layer.js

```diff
@@ -226,7 +529,54 @@ var renderTextLayer = (function renderTextLayerClosure() {
           self._renderTimer = null;
         }, timeout);
       }
-    }
+    },
+    _expandTextDivs: function TextLayer_expandTextDivs(expand) {
+      if (expand) {
+        for (var i = 0, ii = this._textDivs.length - 1; i < ii; i++) {
```

Comment:
I might be misunderstanding, but are you sure about that?
Note that the `endOfContent` div is appended in `text_layer_builder.js` (and only to the DOM, and not `this.textDivs` there) and should thus not affect `this._textDivs` inside `text_layer.js`.